cocos/2d/CCNode.h

```diff
@@ -1881,9 +1894,17 @@ class CC_DLL Node : public Ref
     mutable bool _additionalTransformDirty; ///< transform dirty ?
     bool _transformUpdated;         ///< Whether or not the Transform object was updated since the last frame
 
-    int _localZOrder;               ///< Local order (relative to its siblings) used to sort the node
+    union {
+        struct {
+            int z;
+            unsigned int a;
```

Comment:
it is better to rename them to more meaningful value, for example:

z -> localZorder
a -> orderOfArrival